File: coursework-main/src/main/java/com/example/coursework/newAnalysis/entities/FullName.java
package com.example.coursework.newAnalysis.entities;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
public class FullName {
	private String lastName;
	private String firstName;
	private String patronymic;
}

File: coursework-main/src/main/resources/static/js/entities/FullName.js
export class FullName {
    lastName;
    firstName;
    patronymic;

    constructor(lastName, firstName, patronymic) {
        this.lastName = lastName || "";
        this.firstName = firstName || "";
        this.patronymic = patronymic || "";
    }

    toString() {
        return `${this.lastName} ${this.firstName} ${this.patronymic}`;
    }

}
